cartographer_ros_msgs/msg/metrics/HistogramFamily.msg

```diff
@@ -0,0 +1,17 @@
+# 2018 The Cartographer Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+string name
+string description
+cartographer_ros_msgs/Histogram[] histograms
```

Comment:
This is many msg files for little logic.
Can't we have a MetricFamily.msg that contains Metric.msg, which contains Labels and "one-of" Histogram/Counter/Gauge?
I know there is no "protobuf oneof" in rosmsg.
Should we work around with repeated fields and set only one? What do you think?